cpp/CMakeLists.txt

```diff
@@ -186,7 +186,7 @@ endif()
 include(san-config)
 
 # For any C code, use the same flags.
-set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS}")
+# set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
Good point! I am leaving it in now and remove the C++ specific flags; this might fail on some of the more obscure C compilers but that's probably better than compiling C source files with potentially different optimization flags.